commit f20203aa1ae0c8f2d5dfb0df8d1fd1b0912bc7b1
Author: Russell Brown <russell.brown@joyent.com>
Date:   2019-04-23T16:20:09+01:00 (6 months ago)
    
    MANTA-4237 Bump the version number on node-libmanta for the changes in MANTA-2412

diff --git a/package.json b/package.json
index 62c3efb..cdd4ab1 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
     "name": "libmanta",
     "description": "common functions et al for manta services",
-    "version": "1.2.0",
+    "version": "1.3.0",
     "author": "Joyent (joyent.com)",
     "main": "./lib/index.js",
     "repository": {
